### ISSUE TITLE
Option to omit `eslint-disable`, ... from the header of the generated files

### ISSUE THREAD

**Issue by mirismaili** [ferdikoomen/openapi-typescript-codegen#1539]
I see:
```js
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */
```
on the top of the generated files. These are redundant. Some users don't have these tools in their projects. And others that have one or more of these tools can easily ignore the path of the generated files in the configuration of that tools **IF THEY WANT**. For example, for [ESLint](https://eslint.org/docs/latest/use/configure/ignore):
```json
{
  "ignorePatterns": ["**/generated/**"], // The user can easily ignore the `generated` path IF HE/SHE WANT.
  ...
}
```
Also, this is not a complete list. There are some other tools that aren't ignored here (like [Prettier](https://prettier.io/)). So I think these aren't useful and at least there should be an option to disable it.

**Comment by mrlubos** [comment]
I tend to agree with this @mirismaili. Do you know by any chance why this was introduced?
